src/ui/components/shared/SharingModal/SharingModal.tsx

```diff
@@ -123,9 +127,178 @@ function EnvironmentVariablesRow() {
   return <div className="text-xs">This node recording contains all env variables</div>;
 }
 
+function SharingSection({
+  recording,
+  showEnvironmentVariables,
+  showPrivacy,
+  setShowPrivacy,
+}: {
+  recording: Recording;
+  showEnvironmentVariables: boolean;
+  showPrivacy: boolean;
+  setShowPrivacy: React.Dispatch<React.SetStateAction<boolean>>;
+}) {
+  return (
+    <>
+      <CollaboratorsSection recording={recording} />
+      <section className="flex flex-grow flex-row items-center justify-between space-x-2 bg-menuHoverBgcolor px-8 pt-8">
+        <div className="flex flex-row items-start space-x-3 overflow-hidden">
+          <div className="mt-1 flex h-8 w-8 flex-shrink-0 items-center justify-center rounded-full bg-blue-200 font-bold">
+            <MaterialIcon className="text-blue-600" iconSize="xl">
+              people
+            </MaterialIcon>
+          </div>
+          <div className="flex flex-col space-y-1 overflow-hidden">
+            <div className="font-bold">Privacy Settings</div>
+            <PrivacyDropdown {...{ recording }} />
+            {showEnvironmentVariables ? <EnvironmentVariablesRow /> : null}
+          </div>
+        </div>
+        <CopyButton recording={recording} />
+      </section>
+      <section className="bg-menuHoverBgcolor px-8 pb-6">
+        {!recording.private && recording.operations && (
```

Comment:
Why'd you add the `!recording.private &&` check?

Reply:
This is probably a different PR. good catch